arrows/cuda/integrate_depth_maps.cu

```diff
@@ -126,7 +128,8 @@ __device__ void rayPotential(double realDistance, double depthMapDistance, doubl
   int sign = diff != 0 ? diff / absoluteDiff : 0;
 
   if (absoluteDiff > c_rayPotentialDelta)
-    res = diff > 0 ? 0 : - c_rayPotentialEta * c_rayPotentialRho;
+    res = diff > 0 ? c_rayPotentialEpsilon * c_rayPotentialRho
+                   : - c_rayPotentialEta * c_rayPotentialRho;
```

Comment:
The two lines could read
```suggestion
    res = sign * c_rayPotentialEpsilon * c_rayPotentialRho;
```

Reply:
I don't think this is quite right because in addition to the sign of the value we are also switching between using Epsilon for positive values and Eta for negative values.

Reply:
My bad.